### ISSUE TITLE
Tool crashes while loading a traderconfig

### ISSUE THREAD

**Issue by Dyson08** [rvost/DayzServerTools#1]
Tool crashes without notice when I load a modded traderconfig. Testet it with the original traderconfig which comes with the trder mod and that works. Is there a crash log which give hints on what exactly caused the crash? Cant find the problem within my trader config.

**Comment by rvost** [comment]
@Dyson08 Unfortunately, crash reports not implemented yet. If the config contains errors, you will receive an error message when loading. In case of crash, most likely the problem is on the program side, and not in the configuration file.

Can you provide your modded config so I can locate/fix problem? Sorry for late response.

**Comment by Dyson08** [comment]
Hi @rvost ,

thx for the reply. That is my trader file:
https://pastebin.com/NYz7Qc3R

Kind regards,
Philipp

**Comment by rvost** [comment]
Thanks for sharing, @Dyson08!  

I was able to reproduce the crash, it was caused by an empty buy price on line 5130 `apokot_tackle_box_plano_yellow,*,,142`. It's my fault that I didn't handle this case, but I'm not sure if it's considered a valid config or not. Nevertheless, it will be fixed in next update, which unfortunately will not be released until next Friday or so.

Also note that there are 16 <Cat**a**gory> typos in your file starting at line 4297. You will get error message for this but one at a time.
